OstelcoStyles/OstelcoTextField.swift

```diff
@@ -0,0 +1,150 @@
+//
+//  OstelcoTextField.swift
+//  OstelcoStyles
+//
+//  Created by Ellen Shapiro on 4/29/19.
+//  Copyright © 2019 mac. All rights reserved.
+//
+
+import UIKit
+
+/// Base TextField subclass to facilitate easy IBDesignable subclasses.
+@IBDesignable
+open class OstelcoTextField: UITextField {
+    
+    public var appTextColor: OstelcoColor = .blackForText {
+        didSet {
+            self.textColor = self.appTextColor.toUIColor
+        }
+    }
+    
+    open override var placeholder: String? {
+        didSet {
+            guard let holder = self.placeholder else {
+                self.attributedPlaceholder = nil
+                return
+            }
+            
+            self.attributedPlaceholder = NSAttributedString(string: holder, attributes: [
+                .font: self.appFont.toUIFont,
+                .foregroundColor: self.appPlaceholderColor.toUIColor
```

Comment:
This is the only way to change the font or color of placeholder text on a `UITextField`. Yes, this is silly. Nope, if they haven't changed it yet, they're not gonna.

Reply:
interesting!